tests/testcases.py

```diff
@@ -156,9 +159,13 @@ def run_command(self, args, fake_exit, raise_on_error=True):
         This command raises if the command fails, otherwise it returns the
         stdout generated by the command."""
         with patch('dusty.commands.utils.exec_docker', wraps=self.exec_docker_patch) as fake_exec_docker:
+            fake_exit.side_effect = SysExit('sys.exit called')
             self.fake_exec_docker = fake_exec_docker
             sys.argv = ['dusty'] + args.split(' ')
-            client_entrypoint()
+            try:
+                client_entrypoint()
+            except SysExit:
+                pass
```

Comment:
I need sys.exit to actually "return" (stop executing), but not actually call sys.exit.  I think raising and then catching an exception is the only way to do that...